README.md

```diff
@@ -59,6 +59,33 @@ work on OS X if the databases are all installed using
 To run the linter without running any tests, run `npm run lint`. To
 automatically fix linting errors run `npm run lint-fix`.
 
+### Using Docker for Testing
+
+Running the testsuite on _Jenkins_  is based on docker images. You can also make use of this setup when running tests locally. Scripts are provided for different stages of testing: testing the documentation, running tests against different `nodejs` versions and running tests against different versions of dependencies.
```

Comment:
Please wrap the lines at 72 chars. I know we have talked about doing it differently in the future, but for now the rest of the document uses this convention, so it would be nice if we continued that for the time being 😃